tests/DataFixtures/ORM/resources/fulfilled_cart.yml

```diff
@@ -0,0 +1,138 @@
+Sylius\Component\Core\Model\Order:
+    fulfilled_cart:
+        channel: "@channel_web"
+        addItem: ["@sw_mug_item", "@hard_available_mug_item"]
+        currencyCode: USD
+        localeCode: en_US
+        customer: "@customer_oliver"
+        state: cart
+
+Sylius\Component\Core\Model\OrderItem:
+    sw_mug_item:
+        quantity: 2
+        unitPrice: 2000
+        addUnit: ["@sw_mug_item_unit1", "@sw_mug_item_unit2"]
+        variant: "@mug_sw"
+        order: "@fulfilled_cart"
+    hard_available_mug_item:
+        quantity: 1
+        unitPrice: 2000
+        addUnit: ["@hard_available_mug_item_unit"]
+        variant: "@hard_available_mug"
+        order: "@fulfilled_cart"
+
+Sylius\Component\Core\Model\OrderItemUnit:
```

Comment:
It is just needed to change cart item quantity. Checkout will be checked with request processing